packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -0,0 +1,79 @@
+import { BaseInput } from '../BaseInput';
+import type { BaseInputProps } from '../BaseInput';
+import type { IconComponent } from '~components/Icons';
+import { ChevronDownIcon, ChevronUpIcon } from '~components/Icons';
+import Box from '~components/Box';
+import { VisuallyHidden } from '~components/VisuallyHidden';
+import { getPlatformType } from '~utils';
+type SelectInputProps = Pick<
+  BaseInputProps,
+  | 'label'
+  | 'labelPosition'
+  | 'necessityIndicator'
+  | 'validationState'
+  | 'helpText'
+  | 'errorText'
+  | 'successText'
+  | 'name'
+  | 'isDisabled'
+  | 'isRequired'
+  | 'prefix'
+  | 'suffix'
+  | 'autoFocus'
```

Comment:
placeholder?

Reply:
can't find any of the click handlers? onClick, onFocus, etc?

Reply:
Added `onClick` `onFocus` and `onBlur`.


`onChange` is added in the ActionList PR since that needs a bit special handling.

Reply:
placeholder we're not supporting right now. It is always `Select Option` in the start and then changes into an actual value after selections

Reply:
Also in this case, even if we support it, it will be handled by `defaultValue` prop instead of `placeholder` because text you see inside Select is more of a value and not placeholder.